randovania/games/fusion/logic_database/Sector 1 SRX.txt

```diff
@@ -1077,10 +1077,47 @@ Extra - room_id: [30]
   * Extra - door_idx: (62,)
   > Top of Shaft
       All of the following:
-          Charge Beam or Missiles ≥ 10 or Screw Attack or Combat (Intermediate) or Can Freeze Enemies
           Any of the following:
-              Space Jump
-              Stand On Frozen Enemies (Beginner) and Wall Jump (Beginner) and Can Freeze Enemies
+              # Get past the Rippers in Corridor
+              Morph Ball or Screw Attack
+              # Freeze and do spinjump to glitch through it: https://youtu.be/6TSAshOOI-Q
+              Knowledge (Advanced) and Can Freeze Enemies
```

Comment:
I think this should be bumped down to knowledge intermediate

Reply:
maybe. I feel like most people dont know about it, which fits with toasters anecdotal description:
> Advanced: Most people don't know this, or the logical implications of it suck really bad

Reply:
true but i dont think the logical implications of this suck at all, plus i think most people would think to do this anyway

Reply:
wait ignore, was thinking of the wrong path